src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -3138,6 +3084,8 @@ void PhaseIdealLoop::do_range_check(IdealLoopTree *loop, Node_List &old_new) {
   set_ctrl(opqzm, new_limit_ctrl);
   set_ctrl(iffm->in(1)->in(1), new_limit_ctrl);
   set_ctrl(iffm->in(1), new_limit_ctrl);
+
+  C->print_method(PHASE_AFTER_RANGE_CHECK_ELIMINATION, 4, cl);
```

Comment:
Moved this down because we missed some transformations when having this earlier. Additionally, if there are multiple range checks, we can see the intermediate state for one transformation with the next `PHASE_BEFORE_RANGE_CHECK_ELIMINATION` dump.